packages/cosmic-swingset/lib/ag-solo/vats/issuers.js

```diff
@@ -77,7 +77,7 @@ const fromCosmosIssuerEntries = [
         initialMarginPercent: 150n,
         liquidationMarginPercent: 125n,
         interestRateBasis: 250n,
-        loanFeeBasis: 50n,
```

Comment:
Could this be a ratio instead?

Reply:
The collateralConfig record is used in Bootstrap to build a Ratio, so it could be a ratio already at this point. @michaelfig  do you see any issues with converting to ratios rather than numbers named Percent vs. Basis?

Reply:
> @michaelfig do you see any issues with converting to ratios rather than numbers named Percent vs. Basis?

We don't have the issuers at this point, so the `*Percent` and `*Basis` parameters should be expressed like `loanFeeRatio: [1n, 10000n]` (dimensionless, `[numerator, denominator]`), to which bootstrap will add the dimensions/brands in its `makeRatio` calls.